.ci/test-project/spack_configs/toss_3_x86_64_ib/packages.yaml

```diff
@@ -0,0 +1,6 @@
+packages:
```

Comment:
Can you remove this directory? I am pretty sure it is not used anywhere.

Reply:
its what i used for local testing. i can get rid of it though